src/main/java/org/webjars/taglib/LocateTag.java

```diff
@@ -39,6 +39,15 @@
 	private Iterator<String> result;
 
 	/**
+	 * Whether the path we're looking for do not specify the version number.
```

Comment:
Ah, gotcha. So turns out I misunderstood how this is supposed to be used (and it actually doesn't work like the spring-mvc integration paragraph you mention then, correct?)

To clarify my line of thought: in the spring-mvc doc snippet, it's the spring controller that is doing the job of going version-less to proper path (handling a request that does *not* contain the path, which I thought was to be generated by your change here).
What you're actually proposing here is to have this conversion happen in the taglib itself. Gotcha.

The problem IMO with "the path we're looking for" is that this denotes the *target* path, what you're trying to attain (ie the rewritten path). "Whether version information should be automatically added by this tag" or something similar could be a better alternative, maybe?

I'm sorry I can't help more. Although the original author, it's been a while since I had to use webjars (let alone JSPs) so my memory is a little rusty. The functionality you want to add seems totally legit though (if it did not break backward compatibility, I'd even make it the default), so from my point of view, this PR is greenlit